src/audio_worklet.js

```diff
@@ -97,6 +97,20 @@ function createWasmAudioWorkletProcessor(audioParams) {
         // Reserve space for the output data
         dataPtr += bytesPerChannel * i.length;
       }
+      if (!this.outputViews) {
+        this.outputViews = [];
+        k = outputDataPtr;
+        for (/*which output*/ i of outputList) {
```

Comment:
Can the length of `outputList` be different in each call?

Perhaps `assert(this.outputViews.length == outputList.length)` after this block to be clear?

Reply:
It can grow if other nodes are manually added. I’ll need to write contrived examples to see this, since in the real world I don’t think it will.